app/main/appium.js

```diff
@@ -17,6 +18,9 @@ var batchedLogs = [];
 
 let sessionDrivers = {};
 
+// Delete saved server args, don't start until a server has been started
+settings.delete('SERVER_ARGS');
```

Comment:
if we're not going to `await` this we should at least add a `.catch` to it since it's a promise and we don't want any unhandled rejections.